code/modules/mob/living/carbon/human/diona_gestalt.dm

```diff
@@ -287,6 +289,23 @@
 	visible_message("<span class='warning'>\The [src] quivers slightly, then splits apart with a wet slithering noise.</span>")
 	qdel(src)
 
+/mob/living/carbon/human/diona/handle_speech_problems(var/message, var/verb, var/message_mode)
```

Comment:
This won't work. Humans do not actually use subclasses. They all use base classes. Again, refactor this to be based off of the species datum if you need species specific behaviour.